File: src/main/java/ordilov/lissn/member/application/port/in/AuthCommand.java
package ordilov.lissn.member.application.port.in;

import static ordilov.lissn.member.application.port.in.MemberCommand.RegisterCommand;

import ordilov.lissn.member.domain.AuthInfo.RefreshInfo;
import ordilov.lissn.member.domain.MemberInfo.GetMemberInfo;

public interface AuthCommand {

  GetMemberInfo signUp(RegisterCommand command);

  RefreshInfo refresh(String refreshToken);

}

File: src/main/java/ordilov/lissn/member/application/port/in/AuthCommand.kt
package ordilov.lissn.member.application.port.`in`

import ordilov.lissn.member.domain.GetMemberInfo
import ordilov.lissn.member.domain.RefreshInfo

interface AuthCommand {

    fun signUp(command: RegisterCommand): GetMemberInfo

    fun refresh(refreshToken: String): RefreshInfo

}